neqo-transport/src/connection/tests/recovery.rs

```diff
@@ -203,7 +203,7 @@ fn pto_initial() {
     assert_eq!(delay, INITIAL_PTO * 3);
 }
 
-/// A complete handshake that involves a PTO in the Handshake space.
+/// A complete handshake that involves two PTOs in the Handshake space.
 #[test]
 fn pto_handshake_complete() {
```

Comment:
If I understand correctly, this unit test has been testing time threshold retransmits and not PTOs before. Thus with this pull request, the adjustments below are needed.